ESentryLogRoute.php

```diff
@@ -84,8 +84,15 @@ protected function processLogs($logs)
 
         if ($client) {
             foreach ($logs AS $log) {
+                $strings = explode(PHP_EOL, $log[0]);
+ +              $reg = '/(\#[0-9]+ [\/a-zA-Z0-9.]+?)(\([0-9]+\):\ ?)([^\(]*)(\(.*\))/';
+ +              $replace = '$1$2$3()';
+ +              $result = preg_replace($reg, $replace, $strings);
+ +              $trace = implode(PHP_EOL, $result);
+ +
                 $client->captureMessage(
-                    $log[0],
+ -                  $log[0],
+ +                  $trace,
```

Comment:
https://github.com/getsentry/sentry-php/blob/master/lib/Raven/Client.php
Second param must contain params isn't it?

Reply:
Yes, it must contain parameters.
The error is one, but since the parameters can be different, the errors are defined as different (for example, the user's login)